src/client/components/atoms/burgerButton/index.tsx

```diff
@@ -0,0 +1,52 @@
+import './styles.scss';
```

Comment:
always format and sort imports on every module you work.

`option+shift+f` and `option+shift+o`